app/Http/Controllers/TaskCategoriesController.php

```diff
@@ -41,14 +42,14 @@ public function create()
      * @return \Illuminate\Http\Response
      */
     public function store(TaskCategorySaveRequest $request)
-    {
+    {   
         $taskCategory = new TaskCategory;
         $taskCategory->user_id = $request->user()->id;
         $taskCategory->name = $request->name;
         $taskCategory->save();
-
-        return redirect()->route('taskCategory.index')
-            ->with('success', 'Data Added successfully!');
+        return new TaskCategoryResource(['category'=>$request->name,'msg'=>'category created successfully!!!!']);
```

Comment:
Nopes, this is wrong. You just need to do `return new TaskCategoryResource($taskCategory);`. Why are you trying to pass the message in the response?

Reply:
Ok.